packages/bits/src/lib/image/image.component.ts

```diff
@@ -116,8 +131,11 @@ export class ImageComponent implements OnInit, AfterViewInit {
         let imageHtml: string = "";
         if (_has(image, "code") && _isString(image.code)) {
             imageHtml = image.code;
+            if(image.name && !this.ariaLabel && this.imageName !== image.name) {
+                Promise.resolve().then(_ => this.imageName = image.name);
```

Comment:
Setting an internal property from a getter function is not a good practice, especially when the function is called from the template. That's why you had to wrap it in a promise. I'd prefer to find another way around that.

Overall this function seems quite complex to be called from a template, so updating it wouldn't be a bad thing to do.